lms/static/js/commerce/views/receipt_view.js

```diff
@@ -283,6 +283,38 @@ var edx = edx || {};
 })(jQuery, _, _.str, Backbone);
 
 
+// TODO Move this to a library shared with receipt_view.js
```

Comment:
You have a lot of options. These files are using the ``edx`` namespace convention that we used before RequireJS, but we have now introduced full RequireJS support into the LMS. Given this, you have a few options.

1. The best option would be to convert the two pages to use RequireJS but that is probably more involved than you are looking to do. The benefit is that the pages can be optimized (each bundled into a single minified JS file). @zubair-arbi did this for the logistration pieces recently in this PR: https://github.com/edx/edx-platform/pull/9473, so you can see the patterns there.

2. You can use the namespace to access code loaded from other files. In this file you are defining edx.commerce.ReceiptView, so from any other file you can reference it (as long as it is loaded before the file that needs it). You could introduce a new credit_utils.js file and give it a namespace like edx.commerce.CreditUtils and then access it that way.

3. If you think this is a common utility then you could put it into lms/static/js/utils, and reference it via the namespace again.

I hope this helps. Let me know if you need more details.